Tasks/Gradle/CodeAnalysis/Common/CodeAnalysisResultPublisher.ts

```diff
@@ -111,7 +124,7 @@ export class CodeAnalysisResultPublisher {
             }
         }
 
-        if (buildSummaryLines.length > 0) {
+        if (buildSummaryLines.length > 0 && uploadedArtifacts > 0) { // Do not print this last line if there were no results uploaded
             buildSummaryLines.push('');
             buildSummaryLines.push('Code analysis results can be found in the \'Artifacts\' tab.');
```

Comment:
Fit and finish: there's no point telling the user to find code analysis results in the artifacts tab if, in fact, no artifacts were uploaded.